app/config/FeatureSwitch.scala

```diff
@@ -38,14 +38,30 @@ case class FeatureSwitch(value: Option[Configuration]) {
       case None => Seq()
     }
   }
+
+  def isVersionEnabled(version: String): Boolean = {
+    val versionNoIfPresent: Option[String] =
+      version match {
+        case versionRegex(v) => Some(v)
+        case _               => None
+      }
+
+    val enabled = for {
+      versionNo <- versionNoIfPresent
+      config    <- value
+      enabled   <- config.getOptional[Boolean](s"version-$versionNo.enabled")
+    } yield enabled
+
+    enabled.getOrElse(false)
+  }
 }
 
 sealed case class FeatureConfig(config: Configuration) {
 
   def isSummaryEnabled(source: String, summary: String): Boolean = {
```

Comment:
What is this?

Reply:
Not sure... it is there in all our services... so didn't touch...

Reply:
but never used in our projects...